src/random.cpp

```diff
@@ -436,7 +465,23 @@ static void SeedSlow(CSHA512& hasher) noexcept
     SeedTimestamp(hasher);
 }
 
-static void SeedSleep(CSHA512& hasher)
+/** Extract entropy from rng, strengthen it, and feed it into hasher. */
+static void SeedStrengthen(CSHA512& hasher, RNGState& rng) noexcept
+{
+    // Once per minute, strengthen the hash for 10ms.
+    static std::atomic<int64_t> last_strengthen;
```

Comment:
Initialize?

Reply:
Static variables are always auto-initialized to zero, but for clarity, I've added an explicit zero.